build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+def hadoopAwsVersion = System.getenv('HADOOP_AWS_VER')
+def deltaSparkVersion = System.getenv('DELTA_SPARK_VER')
+def scalaVersion = System.getenv('SCALA_VER')
+
+dependencies {
+    implementation "org.apache.hadoop:hadoop-aws:$hadoopAwsVersion"
+    implementation "io.delta:delta-spark_${scalaVersion}:$deltaSparkVersion"
+    implementation "io.delta:delta-storage:$deltaSparkVersion"
+}
+
+task downloadJars(type: Copy) {
+    def jarDir = file('gradle_jars')
+    from configurations.runtimeClasspath
+    into jarDir
+}
```

Comment:
Does this automatically handle dependencies?

Reply:
I think so. Not exactly sure though.

Reply:
If the delta storage jar is still in the jars list it does

Reply:
:+1: